bin/check_samplesheet.py

```diff
@@ -87,17 +88,28 @@ def check_samplesheet(file_in, file_out):
             "fasta",
         ]
         header = [x.strip('"') for x in fin.readline().strip().split(",")]
-        if header[: len(HEADER)] != HEADER:
+
+        ## Check for missing mandatory columns
+        missing_columns = list(set(HEADER) - set(header))
+        if len(missing_columns) > 0:
             print(
-                "ERROR: Please check samplesheet header -> {} != {}".format(
-                    ",".join(header), ",".join(HEADER)
+                "ERROR: Missing required column header -> {}. Note some columns can otherwise be empty. See pipeline documentation (https://nf-co.re/taxprofiler/usage).".format(
+                    ",".join(missing_columns)
                 )
             )
             sys.exit(1)
 
+        ## Find locations of mandatory columns
+        header_locs = dict()
+        for i in HEADER:
```

Comment:
That's more of a question but if one would like to add an extra column, can one add the name header as well? I am thinking that `HEADER` is already predefined. 

Like:

`sample, run_accession, instrument_platform, fastq_1, fastq_2, fasta, control`

Reply:
HEADER is meant to be the minimum mandatory columns we require. So yes we could extend the list in HEADER to add more mandatory columns if necessary for the pipeline. 

But all other columns are basically 'ignored' for checking here (which is what I'm sort of trying to do in this PR). The output from the tool should only include the columns we need for the pipeline to be executed.